test/data/state-self-employment.js

```diff
@@ -0,0 +1,176 @@
+/* jshint node: true, mocha: true */
+/* jshint -W089 */
+/* jshint -W110 */
+var tito = require('tito');
+var fs = require('fs');
+var path = require('path');
+var yaml = require('js-yaml');
+var assert = require('assert');
+var parse = require('csv-parse');
+var async = require('async')
+
+var _ = require('lodash');
```

Comment:
@shawnbot I used it in a later commit, so unless you think we should avoid having it as a dev dependency, is it ok if we keep it in?